WcaOnRails/app/webpacker/components/Panel/Wrt/EditPerson.jsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import {
+  Button, Form, Icon, Message,
+} from 'semantic-ui-react';
+import _ from 'lodash';
+import { adminUpdatePersonUrl, adminCheckRecordsUrl } from '../../../lib/requests/routes.js.erb';
+import useSaveAction from '../../../lib/hooks/useSaveAction';
+import WcaSearch from '../../SearchWidget/WcaSearch';
+import Loading from '../../Requests/Loading';
+import I18n from '../../../lib/i18n';
+import { genders } from '../../../lib/wca-data.js.erb';
+import 'react-datepicker/dist/react-datepicker.css';
+
+const description = `
+  <p>
+  Choose 'Fix' if you want to replace a person's information in the database.
+  It will modify the Persons table accordingly and the Results table if the person's name is different.
+  This should be used to fix mistakes in the database.
+  </p>
+
+  <p>
+  Choose 'Update' if the person's name or country has been changed.
+  It will add a new entry in the Persons table and make it the current information for that person (subId=1)
+  but it will not modify the Results table so previous results keep the old name.
+  </p>
+`;
+
+const dateFormat = 'YYYY-MM-DD';
+
+const genderOptions = _.map(genders.byId, (gender) => ({
+  key: gender.id,
+  text: gender.name,
+  value: gender.id,
+}));
+
+const countryChangeWarning = `The change you made may have affected national and continental records, be sure to run <a href=${adminCheckRecordsUrl}>check_regional_record_markers</a>.`;
+
+function EditPerson({ countryList }) {
```

Comment:
Same as for genders: You don't need to pass this as a prop, it's available in `wca-data`.